test-zone-data/Connectivity-TP/connectivity04/README.md

```diff
@@ -0,0 +1,249 @@
+# Connectivity04
+
+[This directory](.), i.e. the same directory as this README file, holds
+zone files and `coredns` configuration files for scenarios for test case Connectivity04.
+
+## All message tags
+
+* CN04_EMPTY_PREFIX_SET
+* CN04_ERROR_PREFIX_DATABASE
+* CN04_IPV4_DIFFERENT_PREFIX
+* CN04_IPV4_SAME_PREFIX
+* CN04_IPV4_SINGLE_PREFIX
+* CN04_IPV6_DIFFERENT_PREFIX
+* CN04_IPV6_SAME_PREFIX
+* CN04_IPV6_SINGLE_PREFIX
+
+## All scenarios
+
+Scenario name             | Zone name
+:-------------------------|:---------------------------------------------
+GOOD-1                    | good-1.connectivity04.xa
+GOOD-2                    | good-2.connectivity04.xa
+GOOD-3                    | good-3.connectivity04.xa
+GOOD-4                    | good-4.connectivity04.xa
+EMPTY-PREFIX-SET-1        | empty-prefix-set-1.connectivity04.xa
+EMPTY-PREFIX-SET-2        | empty-prefix-set-2.connectivity04.xa
+EMPTY-PREFIX-SET-3        | empty-prefix-set-3.connectivity04.xa
+ERROR-PREFIX-DATABASE-1   | error-prefix-database-1.connectivity04.xa
+ERROR-PREFIX-DATABASE-2   | error-prefix-database-2.connectivity04.xa
+ERROR-PREFIX-DATABASE-3   | error-prefix-database-3.connectivity04.xa
+IPV4-ONE-PREFIX-1         | ipv4-one-prefix-1.connectivity04.xa
+IPV4-TWO-PREFIXES-1       | ipv4-two-prefixes-1.connectivity04.xa
+IPV6-ONE-PREFIX-1         | ipv6-one-prefix-1.connectivity04.xa
+IPV6-TWO-PREFIXES-1       | ipv6-two-prefixes-1.connectivity04.xa
+IPV4-SINGLE-NS-1          | ipv4-single-ns-1.connectivity04.xa
+IPV6-SINGLE-NS-1          | ipv6-single-ns-1.connectivity04.xa
+DOUBBLE-PREFIX-1          | doubble-prefix-1.connectivity04.xa
+DOUBBLE-PREFIX-2          | doubble-prefix-2.connectivity04.xa
+
+## zonemaster-cli commands and their output for each test scenario
+
+The level (`--level`) must be set to the lowest level of the message tags. For
+this test case `INFO` is the lowest level. It is only meaningful to test the test
+zones with `--test connetivity04`.
+
+The test zones for these scenarios have a dedicated root zone, which means that
+the hint files in the commands below must be used.
+
+All commands are run from the same directory as this file is in.
+
+**The commands below were run before the necessary update of the implementation
+of the test case.**
```

Comment:
Maybe I'm missing something, but wouldn't it be better to specify what the output is supposed to look like in a README file in the specifications? Copies of whatever the outputs looked like at some point in time is something I'd rather expect to find in a test report document.

And I don't see the purpose of including timestamps, level and test case here in the specification. Even the message tag arguments seem excessive from my understanding.

Reply:
The README file has been renamed and its role has been clarified. A new README has been created.